modules/sharedaddy/sharing.js

```diff
@@ -21,7 +21,7 @@ if ( sharing_js_options && sharing_js_options.counts ) {
 				requests = {
 					// LinkedIn actually gets the share count for both the http and https version automatically -- so we don't need to do extra magic
 					linkedin: [
-							'https://www.linkedin.com/countserv/count/share?format=jsonp&callback=updateLinkedInCount&url=' +
+						'https://www.linkedin.com/countserv/count/share?format=jsonp&callback=WPCOMSharing.update_linkedin_count&url=' +
```

Comment:
For reference, in case anyone looks at it like I did, it looks like this was a revert of #3795, as the change was no longer needed: r136117-wpcom / p47NkD-zd-p2 

@georgestephanis The `updateLinkedInCount` var (line 118) can now be removed in Jetpack as well, I think.